contrib/segyconverter/segy_convert_sample.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
What's the default kernel here? Suggest you set seismic-interpretation

Reply:
fixed in last commit